keyboards/qwertykeys/qk100/ansi/config.h

```diff
@@ -0,0 +1,53 @@
+/* 
+Copyright 2023 Qwertykeys
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define DYNAMIC_KEYMAP_LAYER_COUNT 2  
+
+#    define RGBLIGHT_LAYERS                   
+#    define RGBLIGHT_LAYERS_OVERRIDE_RGB_OFF 
+#    define RGBLIGHT_LAYERS_RETAIN_VAL      
+#    define RGBLIGHT_DEFAULT_VAL 10
+#    define RGBLIGHT_DEFAULT_MODE RGBLIGHT_MODE_RAINBOW_SWIRL + 5
+#    define RGBLIGHT_SLEEP
+/*== customize breathing effect ==*/
+#    define RGBLIGHT_BREATHE_TABLE_SIZE 128
+#    define RGBLIGHT_EFFECT_BREATHE_CENTER 1.0
+#    define RGBLIGHT_EFFECT_BREATHE_MAX    60
+
+
+/* RGB matrix */
+#ifdef RGB_MATRIX_ENABLE
+// i2c setting
+#define USE_I2CV1
+#define I2C1_SCL 8  
+#define I2C1_SDA 9  
```

Comment:
`USE_I2CV1` is conditionally defined for this MCU, and `I2C1_SDA/SCL` defines do not exist.
```suggestion
```